packages/relay/src/lib/db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository.ts

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Please add tests similar to the ones in `ethAddressHbarSpendingPlanRepository.spec.ts`